static/openapi/v2.0/ProducerAuthorityList.yaml

```diff
@@ -0,0 +1,20 @@
+type: array
+description: Array that has as first element the version of the authority json objects and then the objects themselves.
```

Comment:
Consider:
"Array with the version of the authority JSON objects as first element, then the objects themselves."